content/en/docs/sdk-utilities/s3.md

```diff
@@ -433,3 +433,31 @@ fmt.Printf("Bucket %s is in %s region\n", bucket, region)
 If `GetBucketRegion` is not able to resolve the location of a Bucket, the function returns a
 [BucketNotFound]({{% apiref "feature/s3/manager#BucketNotFound" %}}) error type as shown in the example.
 
+
+# Unseekable Streaming Input
+
+For API operations like `PutObject`, the {{% alias service=S3 %}} client expects the value of the `Body` input parameter to implement the [io.Seeker](https://pkg.go.dev/io#Seeker) interface by default. The `io.Seeker` interface is used by the client to determine the length of the value to upload, and to compute payload hash for the [request signature](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html). If the `Body` input parameter value does not implement `io.Seeker`, your application will receive an error.
```

Comment:
> For API operations like `PutObject`

Are there other operations? If so, should we have a list here.

Reply:
I left that open here since more operations could be added and this doc would get outdated. There is at least `PutObject` and `UploadPart` operations this applies to and will list both here.